1. Select basics/Task 1 Select/Result/Task1_Natalli_Sholomitskaya.sql

```diff
@@ -0,0 +1,29 @@
+select * from employees;
+
+select department_id, department_name, manager_id as "MNG"
+from departments;
+
+select employee_id, first_name, last_name, phone_number, department_id
+from employees;
+
+select first_name, last_name, hire_date, salary, salary + salary * 20 / 100 as "ANNUAL_SAL"
```

Comment:
That is an incorrect calculation. you can do (salary + salary * 20 )/ 100 or **salary*1.2**